src/commands/git.rs

```diff
@@ -736,36 +737,52 @@ fn cmd_git_push(
                 )?;
             }
         }
+
+        for commit in revision_commits {
+            for (branch_name, branch_target) in
+                find_branches_targeting(repo.view(), &RefTarget::Normal(commit.id().clone()))
+            {
+                if !seen_branches.insert(branch_name.clone()) {
+                    continue;
+                }
+                let push_action = classify_branch_push_action(branch_target, &remote);
+                match push_action {
+                    BranchPushAction::AlreadyMatches => {}
+                    BranchPushAction::LocalConflicted => {}
+                    BranchPushAction::RemoteConflicted => {}
```

Comment:
This can be replaced with an OR pattern. 
```rust
match push_action {
  BranchPushaction::AlreadyMatches | BranchPushaction::LocalConflicted
  | BranchPushaction::RemoteConflicted => {}
  //...
``` 

}

Reply:
Good point.